looper/models.py

```diff
@@ -934,11 +934,27 @@ def add_sample_sheet(self, csv=None):
                             # so we don't re-derive them later.
                             merged_cols = {
                                     key: "" for key in merge_rows.columns}
-                            for row in merge_rows.index:
+                            for row in range(len(merge_rows.index)):
```

Comment:
Just one comment on this: while this does fix the issue of passing an integer to `.iloc` the better option would be to keep iterating through the index itself and pass the values to `.loc` which accommodates all types of indexes. As far as I can see that would never raise an error.